browser/utils.js

```diff
@@ -1,13 +1,19 @@
 export function putUserOnDOM (user) {
   const scene = document.getElementById('scene');
-  const avatar = document.createElement('a-entity');
-  scene.appendChild(avatar);
-  avatar.setAttribute('id', user.id);
-  avatar.setAttribute('geometry', 'primitive', 'box');
-  avatar.setAttribute('material', 'color', user.color);
-  avatar.setAttribute('position', `${user.x} ${user.y} ${user.z}`);
-  avatar.setAttribute('rotation', `${user.xrot} ${user.yrot} ${user.zrot}`);
-  return avatar;
+  const head = document.createElement('a-minecraft');
+  const body = document.createElement('a-minecraft');
+  scene.appendChild(head);
+  scene.appendChild(body);
+  head.setAttribute('id', user.id);
+  body.setAttribute('id', `${user.id}-body`);
+  head.setAttribute('minecraft-nickname', user.color);
+  head.setAttribute('minecraft', 'skinUrl: ../../images/3djesus.png;');
```

Comment:
is this just getting the head? just curious how that's working since for body you had to say component:body